Wellthier/Models/Workout.h

```diff
@@ -20,6 +21,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong) NSMutableArray *exercises;
 
 + (void) postUserWorkout: ( UIImage * _Nullable )image withTitle: ( NSString * _Nullable )title withCompletion: (PFBooleanResultBlock  _Nullable)completion;
++ (void) updateUserWorkout: ( Workout * _Nullable )workout withExercise: ( Exercise * _Nullable )exercise withCompletion: (PFBooleanResultBlock  _Nullable)completion;
++ (void) deleteExerciseFromWorkout: ( Workout * _Nullable )workout withExercise: ( Exercise * _Nullable )exercise withCompletion: (PFBooleanResultBlock  _Nullable)completion;
```

Comment:
avoid additional space in arguments
+ (void) deleteExerciseFromWorkout: (Workout * _Nullable)workout withExercise: (Exercise * _Nullable)exercise withCompletion: (PFBooleanResultBlock  _Nullable)completion;